e2e/rbd.go

```diff
@@ -1193,6 +1194,23 @@ var _ = Describe("RBD", func() {
 					}
 				}
 
+				// Resize Filesystem PVC and check application directory size
+				// Resize 0.3.0 is only supported from v1.15+
+				if k8sVersionGreaterEquals(f.ClientSet, 1, 15) {
+					err = deleteResourceWithIgnoreNotFound(pvcPath)
+					if err != nil {
+						e2elog.Failf("failed to delete filesystem PVC with error %v", err)
+					}
+					err = deleteResourceWithIgnoreNotFound(appPath)
+					if err != nil {
+						e2elog.Failf("failed to delete app with error %v", err)
+					}
```

Comment:
we don't need this one,  the resources will be cleaned up in the previous cases. this can be removed

Reply:
Thanks! Done.